var/spack/repos/builtin/packages/netlib-scalapack/package.py

```diff
@@ -97,6 +142,7 @@ def cmake_args(self):
             c_flags.append("-Wno-error=implicit-function-declaration")
 
         options.append(self.define("CMAKE_C_FLAGS", " ".join(c_flags)))
+        options.append(self.define("CMAKE_FFLAGS", " ".join(fflags)))
```

Comment:
Usually CMake expects `CMAKE_Fortran_FLAGS` (where "Fortran" is intentionally not all upper-case). I can't find anything about `CMAKE_FFLAGS` in the CMake docs. Could you set `CMAKE_Fortran_FLAGS`?

Reply:
@scheibelp makes sense and in fact it's consistent with line 120 -- for whatever reason the variable I used was picked up correctly in my environment, but happy to switch to the more generally accepted one. Thanks!